cmd/kubeadm/app/phases/apiconfig/setupmaster.go

```diff
@@ -49,6 +50,8 @@ func findMyself(client *clientset.Clientset) (*v1.Node, error) {
 }
 
 func attemptToUpdateMasterRoleLabelsAndTaints(client *clientset.Clientset) error {
+	validate.WaitForNode(client)
```

Comment:
Earlier (before #43881) this was tested during client create, so this was even earlier.
Without this - list queried in `findMyself` can be empty - what leads to fail described in https://github.com/kubernetes/kubernetes/pull/43881#issuecomment-291195835